sites/kit.svelte.dev/vite.config.js

```diff
@@ -26,10 +26,101 @@ const config = {
 				const ext = path.extname(url.pathname);
 				const params = new URLSearchParams();
 				params.set('format', 'avif;webp;' + fallback[ext]);
-				if (!params.has('meta') && !params.has('metadata') && !params.has('source') && !params.has('srcset') && !params.has('url')) {
+				if (
+					!params.has('meta') &&
+					!params.has('metadata') &&
+					!params.has('source') &&
+					!params.has('srcset') &&
+					!params.has('url')
+				) {
 					params.set('picture', true);
 				}
 				return params;
+			},
+			extendOutputFormats: (builtins) => {
+				/** @type {import('vite-imagetools').OutputFormat} */
+				function generate() {
```

Comment:
This looks like it was just for testing and could be imported from the image package